sympy/functions/combinatorial/factorials.py

```diff
@@ -209,10 +209,15 @@ class subfactorial(CombinatorialFunction):
     It can also be written as int(round(n!/exp(1))) but the recursive
     definition with caching is implemented for this function.
 
-    This function is generalized to noninteger arguments [2]_ as
+    This function is generalized [2]_ as
 
     .. math:: !x = \Gamma(x + 1, -1)/e
 
+    This function leaves non-integer arguments unevaluated.  The above formula
```

Comment:
Perhaps continuing the previous line:
"which is valid for non-negative integers x."

instead of
"This function..."